src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
```

Comment:
@jnm2 - if you're still interested in this - would you mind adding a separate issue? Don't want this to be lost in amongst this PR - I think it's worth saving, but let's get this one merged!